SiennaOpenAPIModels.jl/src/json_to_sienna/common.jl

```diff
@@ -14,10 +14,210 @@ function get_bustype_enum(bustype::String)
     end
 end
 
+function get_prime_mover_enum(prime_mover::String)
```

Comment:
We can probably replace the enum conversions with IS.deserialize if we make InfrastructureSystems a dependency.